### ISSUE TITLE
PPO算法训练和验证以及测试过程中action_select疑问

### ISSUE THREAD

**Issue by HUSTHY** [datawhalechina/easy-rl#81]
PPO算法中，训练和验证阶段 行动选择都是同一种方案，都是通过actor网络输出的logits概率建立分布后，进行抽样得到的。
def choose_action(self, state):
        state = torch.tensor([state], dtype=torch.float).to(self.device)
        dist = self.actor(state)
        value = self.critic(state)
        action = dist.sample()
        probs = torch.squeeze(dist.log_prob(action)).item()
        if self.continuous:
            action = torch.tanh(action)
        else:
            action = torch.squeeze(action).item()
        value = torch.squeeze(value).item()
        return action, probs, 
是不是验证阶段应该不一样才对，直接取概率最大的那个行动？
dist = self.actor(state)
action = torch.argmax(dist,dim=1)

**Comment by johnjim0816** [comment]
> PPO算法中，训练和验证阶段 行动选择都是同一种方案，都是通过actor网络输出的logits概率建立分布后，进行抽样得到的。 def choose_action(self, state): state = torch.tensor([state], dtype=torch.float).to(self.device) dist = self.actor(state) value = self.critic(state) action = dist.sample() probs = torch.squeeze(dist.log_prob(action)).item() if self.continuous: action = torch.tanh(action) else: action = torch.squeeze(action).item() value = torch.squeeze(value).item() return action, probs, 是不是验证阶段应该不一样才对，直接取概率最大的那个行动？ dist = self.actor(state) action = torch.argmax(dist,dim=1)

基于策略梯度的算法如果不加额外探索策略的话，训练和测试过程中选择的动作基本都一样，只有dqn这种用e-greedy策略会不一样

**Comment by HUSTHY** [comment]
已收到！谢谢！                          ——黄洋
